model/model.py

```diff
@@ -43,37 +44,45 @@ def init_surface(self, kind: str):
             self.__update_neighbours()
             self.__clear_and_update_active_cells_border()
             return
-            
-        raise NotImplementedError
 
+        raise NotImplementedError
 
     def update_current(self, directions, speed):
         for i, row in enumerate(self.cells):
             for j, cell in enumerate(row):
                 direction = directions[i][j]
-                neighbour = 0
-                if direction[0] == -1 and direction[1] == -1:
-                    neighbour = 0
-                if direction[0] == -1 and direction[1] == 0:
-                    neighbour = 1
-                if direction[0] == -1 and direction[1] == 1:
-                    neighbour = 2
-                if direction[0] == 0 and direction[1] == -1:
-                    neighbour = 3
-                if direction[0] == 0 and direction[1] == 0:
-                    neighbour = -1
-                if direction[0] == 0 and direction[1] == 1:
-                    neighbour = 4
-                if direction[0] == 1 and direction[1] == -1:
-                    neighbour = 5
-                if direction[0] == 1 and direction[1] == 0:
-                    neighbour = 6
-                if direction[0] == 1 and direction[1] == 1:
-                    neighbour = 7
-
-                cell.update_current_direction(neighbour)
+                neighbours = self._get_neighbours(direction)
+                cell.update_current_direction(neighbours)
```

Comment:
jeśli przeniesiem __get_neighbours do cellki to to będzie można dać w jedną funckję